pkg/loop/telem.go

```diff
@@ -149,6 +149,7 @@ func (config TracingConfig) NewSpanExporter() (sdktrace.SpanExporter, error) {
 	traceExporter, err := otlptracegrpc.New(ctx,
 		otlptracegrpc.WithGRPCConn(conn),
 		otlptracegrpc.WithHeaders(config.AuthHeaders),
+		otlptracegrpc.WithCompressor("gzip"),
```

Comment:
Use `gzip.Name` for imported artifacts. gzip package contains init that runs only if pkg is imported. Without it compression might not work